ansible_collections/arista/avd/molecule/eos_designs_unit_tests/documentation/devices/DC1-BL2A.md

```diff
@@ -438,13 +438,13 @@ interface Vlan350
 
 #### UDP port: 4789
 
-#### VLAN to VNI Mappings
```

Comment:
This looks like stuff from PR #1121, possibly the documentation wasn't updated before it got merged?

Reply:
I have merged PR #1171 to resync all artifacts in Molecule